src/N3Lexer.js

```diff
@@ -302,12 +302,20 @@ export default class N3Lexer {
       case ')':
       case '{':
       case '}':
+        if (input.length > 1 && input[1] === '|') {
+          type = '{|', matchLength = 2;
+          break;
+        }
         if (!this._lineMode) {
           matchLength = 1;
           type = firstChar;
         }
         break;
-
+      case '|':
+        if (input.length > 1 && input[1] === '}') {
```

Comment:
Idem; should be inconclusive if input length smaller than 2. And we should test a split `|` and `}` (can be the same test as the split starting token).

Reply:
Likewise we should also be testing a split in `<` and `<` in the start of quoted `<<` statements I guess?

Reply:
Indeed; I see that's covered there.